flow/designs/ihp-sg13g2/gcd/config.mk

```diff
@@ -0,0 +1,13 @@
+export DESIGN_NAME = gcd
+export PLATFORM    = ihp-sg13g2
+
+export VERILOG_FILES = ./designs/src/$(DESIGN_NICKNAME)/gcd.v
+export SDC_FILE      = ./designs/$(PLATFORM)/$(DESIGN_NICKNAME)/constraint.sdc
+
+# Adders degrade GCD
```

Comment:
Is this true in this pdk?

Reply:
We do not provide adders cells so I do not think it is relevant 
```
# Adders degrade GCD
export ADDER_MAP_FILE :=
```
removed